crates/types/src/serde_helpers.rs

```diff
@@ -9,15 +9,23 @@ pub mod rpc_sierra_contract_class_to_sierra_contract_class {
     {
         let mut json_obj = serde_json::Value::deserialize(deserializer)?;
         // Take the inner part of the string value which is expected to be a JSON array and replace
-        // it
+        // it with the deserialized value.
+        // If for some reason the abi field is empty string, remove it from collection
```

Comment:
Why would it be an empty string?

Reply:
Its a possible situation, in the tests I encountered it

Reply:
alright, resolving